packages/ckbtc/src/types/bitcoin.params.ts

```diff
@@ -14,6 +18,24 @@ export const toGetUtxosParams = ({
   ...rest
 }: GetUtxosParams): get_utxos_request => ({
   filter: toNullable(filter),
-  network: network === "testnet" ? { testnet: null } : { mainnet: null },
+  network: parseBitcoinNetwork(network),
+  ...rest,
+});
+
+export type GetBalanceParams = Omit<
+  get_balance_request,
+  "network" | "min_confirmations"
+> & {
+  network: BitcoinNetwork;
+  minConfirmations?: number;
```

Comment:
I would stick to the naming `min_confirmations` for consistency reason given that `GetUtxosParams` uses `min_confirmations`

Reply:
you sure? it was more in the idea of "use snake-case in backend, use camel-case on frontend", but if we don't have that limitation, I adjust

Reply:
I'm not sure neither but, if we modify it, then we have to modify the other and it's a breaking change, that's why I though we should stick to the existing pattern, for the best or worst.

Reply:
cool! will do!